mix.exs

```diff
@@ -20,7 +20,7 @@ defmodule Hexpm.MixProject do
   def application() do
     [
       mod: {Hexpm.Application, []},
-      extra_applications: [:logger, :runtime_tools, :os_mon]
+      extra_applications: [:logger, :runtime_tools, :os_mon, :hex]
```

Comment:
this is to avoid this warning:

```
warning: Hex.version/0 defined in application :hex is used by the current application but the current application does not directly depend on :hex. To fix this, you must do one of:

  1. If :hex is part of Erlang/Elixir, you must include it under :extra_applications inside "def application" in your mix.exs

  2. If :hex is a dependency, make sure it is listed under "def deps" in your mix.exs

  3. In case you don't want to add a requirement to :hex, you may optionally skip this warning by adding [xref: [exclude: Hex] to your "def project" in mix.exs

  lib/hexpm/repository/resolver.ex:39: Hexpm.Repository.Resolver.resolve_deps/1
```

feels a bit dirty though.